src/main/java/hudson/plugins/analysis/views/AbstractAnnotationsDetail.java

```diff
@@ -78,7 +79,7 @@ public String getHeader() {
      *
      * @return the owner
      */
-    public final AbstractBuild<?, ?> getOwner() {
+    public final Run<?, ?> getOwner() {
```

Comment:
another compatibility issue. It could be solved by a new method even if it's not elegant